flux/flux-core/src/main/java/org/apache/storm/flux/Flux.java

```diff
@@ -52,17 +52,22 @@
 public class Flux {
     private static final Logger LOG = LoggerFactory.getLogger(Flux.class);
 
+    @Deprecated
```

Comment:
Given that current scripts to start a Flux topology will have to be changed to make this work, is there any value to preserving these options? If someone is using one of these, most likely their script will end up failing or doing the wrong thing.

Reply:
IMHO this it is acceptable for a major release.  Projects need a drastic cleanup once in a while.

Reply:
Sure, what I meant was if we're going to break the API anyway I think it makes sense to remove the options entirely, rather than deprecating them.

Reply:
Got you! Your comment was pretty clear, I just need another coffee :-)